geode-core/src/test/java/org/apache/geode/management/internal/cli/domain/RegionDescriptionJUnitTest.java

```diff
@@ -0,0 +1,312 @@
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.management.internal.cli.domain;
+
+import static org.apache.geode.cache.DataPolicy.NORMAL;
+import static org.apache.geode.cache.Scope.DISTRIBUTED_ACK;
+import static org.apache.geode.cache.Scope.LOCAL;
+import static org.apache.geode.management.internal.cli.domain.RegionDescription.findCommon;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.when;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.json.JSONObject;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import org.apache.geode.management.cli.Result.Status;
+import org.apache.geode.management.internal.cli.commands.DescribeRegionCommand;
+import org.apache.geode.management.internal.cli.result.CommandResult;
+import org.apache.geode.test.junit.categories.UnitTest;
+
+@Category(UnitTest.class)
+public class RegionDescriptionJUnitTest {
+  private static final String evictionKeyShared = "sharedEvictionKey";
+  private static final String partKeyShared = "sharedPartitionKey";
+  private static final String regKeyShared = "sharedRegionKey";
+  private static final String evictionValueShared = "sharedEvictionValue";
+  private static final String partValueShared = "sharedPartitionValue";
+  private static final String regValueShared = "sharedRegionValue";
+
+  private static final String evictionKeyA = "uniqueEvictionKey_A";
+  private static final String partKeyA = "uniquePartitionKey_A";
+  private static final String regKeyA = "uniqueRegionKey_A";
+  private static final String evictionValueA = "uniqueEvictionValue_A";
+  private static final String partValueA = "uniquePartitionValue_A";
+  private static final String regValueA = "uniqueRegionValue_A";
+
+  private static final String evictionKeyB = "uniqueEvictionKey_B";
+  private static final String partKeyB = "uniquePartitionKey_B";
+  private static final String regKeyB = "uniqueRegionKey_B";
+  private static final String evictionValueB = "uniqueEvictionValue_B";
+  private static final String partValueB = "uniquePartitionValue_B";
+  private static final String regValueB = "uniqueRegionValue_B";
+
+  public static final String regionName = "mockRegion1";
+
+  private static final DescribeRegionCommand command = spy(DescribeRegionCommand.class);
```

Comment:
It's not necessary to  spy this, just define "command = new DescribeRegionCommand()".

The spy is really only necessary if you're specifying some test action using "when()" or using "verify" to makesure specific calls are made.

Reply:
Thanks.  Still learning the ropes when it comes to Mockito.

I've refactored these into its own class and extended the mocking around DescribeRegionCommand, since it's really not part of the RegionDescription testing anyway.